src/nni_manager/rest_server/restValidationSchemas.ts

```diff
@@ -51,6 +51,7 @@ export namespace ValidationSchemas {
                 command: joi.string().min(1),
                 virtualCluster: joi.string(),
                 shmMB: joi.number(),
+                nasMode: joi.string().valid('classic_mode', 'enas_mode', 'oneshot_mode'),
```

Comment:
what's the meaning of "enas-mode" or "one-shot mode"? do we have any doc about that?

Reply:
is this nasMod field optional?

Reply:
Originally I set it to be optional so that the previous code can run. I'll check again.

Reply:
> what's the meaning of "enas-mode" or "one-shot mode"? do we have any doc about that?

A simple doc has been post in the conversation overview of this PR. I'm now modifying the detailed document.